releng/org.pitest.pitclipse.target/org.pitest.pitclipse.target.target

```diff
@@ -10,7 +10,7 @@
 			<unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.swtbot.eclipse.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.swtbot.feature.group" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/releases/oxygen"/>
+			<repository location="https://download.eclipse.org/releases/photon"/>
```

Comment:
That's a breaking change, isn't it? With this target platform Pitlcipse won't run on Oxygen-based products anymore, right?

Reply:
No, not at all: we need photon to run the UI tests with Java 11 (in turn, required by SonarQube analysis). There was a bug in Oxygen that prevented UI tests to be run by Tycho and Java 11.
Besides that, all the other version ranges of our deployed features haven't changed, so Pitclipse could still be installed in Oxygen-based products (probably even in older versions).
It's just that we test it with Photon.